config/src/config/api_config.rs

```diff
@@ -42,6 +42,10 @@ pub struct ApiConfig {
 
     /// Max gas unit for view function.
     pub max_gas_view_function: u64,
+
+    /// Performance functionality
```

Comment:
This doc comment is on the below value, let's please add a doc comment per each which describes what the value changes

Reply:
Aah, thanks. Copy-pasta from the above comment around pagination. Fixed that comment, too.